pkg/health/server/server.go

```diff
@@ -27,10 +27,10 @@ import (
 	"github.com/cilium/cilium/pkg/health/defaults"
 	"github.com/cilium/cilium/pkg/lock"
 	"github.com/cilium/cilium/pkg/logging"
+	"github.com/cilium/cilium/pkg/logging/logfields"
 
 	"github.com/go-openapi/loads"
-	flags "github.com/jessevdk/go-flags"
-	"github.com/sirupsen/logrus"
+	"github.com/jessevdk/go-flags"
```

Comment:
Is this import still used? How can you change the import’s name without changing any usage of it?

Reply:
I saw this too, and was surprised, but it gets imported correctly and is marked as redundant by my IDE:

<img width="393" alt="screen shot 2018-07-23 at 10 56 39 pm" src="https://user-images.githubusercontent.com/3858271/43119568-c05c5dfa-8ecb-11e8-92db-c5591b5cf3af.png">

Reply:
:( I think go-flags imports itself as flags and this makes me sad